### PR TITLE
Implemented min in paddle.tensor

### LINKED ISSUE #17679: min

### PR DESCRIPTION
**Pull request by Law47**
Fixes #17679

### CODE REVIEW

**Review by sherry30 — CHANGES_REQUESTED** [review]
Hi,
Can you remove the changes from the .idea files.
Also you have to add a test for the function that you've added. You can go through our docs [here](https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends_tests.html) to get an idea how you would do that. 

Thanks

**Review by sherry30 — CHANGES_REQUESTED** [review]
Hi,

The changes in the .idea folder are still there. Can you remove those.
I think you've added the min function twice by accident, can you remove one implementation.'

Thanks

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py [hunk 1]
```diff
@@ -1816,3 +1816,44 @@ def test_paddle_deg2rad(
...
         on_device=on_device,
     )
+
+# min
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="paddle.to_tensor",
+    method_name="min",
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=st.one_of(helpers.get_dtypes("float", "integer")),
```
Comment (sherry30) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py:1826]: You can add `helpers.get_dtypes("numeric")` instead

File: ivy/functional/frontends/paddle/tensor/tensor.py [hunk 2]
```diff
@@ -355,3 +355,10 @@ def max(self, axis=None, keepdim=False, name=None):
     @with_unsupported_dtypes({"2.5.0 and below": ("float16", "bfloat16")}, "paddle")
     def deg2rad(self, name=None):
         return ivy.deg2rad(self._ivy_array)
+
+    @with_supported_dtypes(
+        {"2.5.0 and below": ("float16", "float32", "float64", "int32", "int64")},
```
Comment (sherry30) [inline ivy/functional/frontends/paddle/tensor/tensor.py:360]: `float16` is not supported for this

File: ivy/functional/frontends/paddle/tensor/tensor.py [hunk 3]
```diff
@@ -355,3 +355,10 @@ def max(self, axis=None, keepdim=False, name=None):
     @with_unsupported_dtypes({"2.5.0 and below": ("float16", "bfloat16")}, "paddle")
     def deg2rad(self, name=None):
         return ivy.deg2rad(self._ivy_array)
+
+    @with_supported_dtypes(
+        {"2.5.0 and below": ("float16", "float32", "float64", "int32", "int64")},
+        "paddle",
+    )
+    def min(self, axis=None, keepdim=False, name=None):
+        return ivy.min(self._ivy_array, axis=axis, keepdim=keepdim)
```
Comment (sherry30) [inline ivy/functional/frontends/paddle/tensor/tensor.py:364]: `ivy.min` takes `keepdims` with the "s".

**Review by sherry30 — CHANGES_REQUESTED** [review]
minor changes,
other than that works fine

Thanks

File: ivy/functional/frontends/paddle/tensor/tensor.py [hunk 4]
```diff
@@ -388,3 +388,10 @@ def isclose(self, y, rtol=1e-05, atol=1e-08, equal_nan=False, name=None):
     def floor_divide(self, y, name=None):
         y_ivy = y._ivy_array if isinstance(y, Tensor) else _to_ivy_array(y)
         return ivy.floor_divide(self._ivy_array, y_ivy)
+
+    @with_supported_dtypes(
+        {"2.5.0 and below": ("float32", "float64", "int32", "int64")},
+        "paddle",
+    )
+    def min(self, axis=None, keepdims=False, name=None):
```
Comment (sherry30) [inline ivy/functional/frontends/paddle/tensor/tensor.py:396]: Its still `keepdim` for paddle's `min` function.
only ivy's function has `keepdims`

**Review by sherry30 — CHANGES_REQUESTED** [review]
minor change

**Comment by sherry30** [comment]
ivy-gardener

**Review by sherry30 — CHANGES_REQUESTED** [review]
lgtm!

Can you resolves the conflicts, I'll merge it as soon as you do. Also feel free to ask for a review when you're done making changes. 😄 

Thanks

**Comment by Law47** [comment]
Hello, I have resolved the conflicts. Please review my PR.

**Comment by sherry30** [comment]
ivy-gardener

**Pull request merged**